packages/grid/_modules_/grid/locales/zhCN.ts

```diff
@@ -0,0 +1,106 @@
+import { enUS as enUSCore } from '@material-ui/core/locale';
```

Comment:
The core has a `zhCN` local, we should probably use it instead of `enUS`

Reply:
Ahh~, it's a mistake. Thank you very much.